packages/zoe/src/zoeService/startInstance.js

```diff
@@ -18,11 +18,13 @@ import {
   InstanceAdminI,
   InstanceAdminShape,
 } from '../typeGuards.js';
+import '../internal-types.js';
+import './internal-types.js';
```

Comment:
are these strictly necessary? Please try to remove. But this appears to be net forward progress so I won't block on it